man/zetta_colors.5.md

```diff
@@ -269,14 +269,14 @@ So setting `EXA_COLORS="reset:*.txt=31"` will highlight only text files; setting
 AUTHOR
 ======
 
-exa is maintained by Benjamin ‘ogham’ Sago and many other contributors.
+Zetta is a fork of exa by Benjamin ‘ogham’ Sago and many other contributors.
```

Comment:
I'm not really sure about a better wording but I'd prefer something that communicates clearer that _exa_ is by ogham.
I think one could interpret this phrase also as _the fork_ is by ogham.

Maybe
```suggestion
Zetta is a fork of exa by Benjamin ‘ogham’ Sago maintained by Denis ’syphar’ Cornehl and many other contributors.
```